packages/terra-status-view/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ ChangeLog
 
 Unreleased
 ----------
+### Changed
+* `<div>` tag replaced with `<p>` in message content.
```

Comment:
can we specify that this change is for a11y?

Reply:
Updated here 8f59698af7fd91b2b9c32979400bc90749734ce6